### ISSUE TITLE
Bubbles not working on Samsung A55 5G

### ISSUE THREAD

**Issue by burtek** [signalapp/Signal-Android#13542]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Bubbling does not work on A55 5G with Android 14 / OneUI 6.1 / April 2024 Security Patch

### Steps to reproduce
- no bubbles appear when getting message
- using "Open bubble" from chat's menu only shows empty notification, no bubble appears

Reopening after #13325 as per https://github.com/signalapp/Signal-Android/issues/13325#issuecomment-2067022409 as it's still not working with April Security patches for me.

### Screenshots
App, notification appears. No bubbble | Notifications list
---|---
![signal-2024-04-30-144634_002](https://github.com/signalapp/Signal-Android/assets/2940958/fb69ceba-70ed-447d-ab67-975126142d01) | ![signal-2024-04-30-144647_002](https://github.com/signalapp/Signal-Android/assets/2940958/ff2706f4-57b1-4eed-9bb8-73c6f179588f)

### Device info
**Device:** Samsung Galaxy A55 5G
**Android version:** Android 14, April Security Patch
**Signal version:** 7.5.2 (edit: same with 7.6.2)

### Link to debug log
https://debuglogs.org/android/7.5.2/baa105db6e3c9d89607ad3ebeeb6c0d9a3c316543db42ff19224316f4053441d - right after clicking "Open bubble" option

**Comment by oliszymanski** [comment]
I don't think that this is an app issue but an Android setting that you can turn on/off if you like. Here's what I found and could help you:
https://www.reddit.com/r/signal/comments/ntvwvw/disable_chat_bubbles/?rdt=59458

My answer is also based on this post oczywiście 😁

**Comment by burtek** [comment]
@oliszymanski tried that and multiple other things from the Internet already, nothing works. Seems to me like my phone got no update from Samsung unbanning Signal bubbles

**Comment by alex-signal** [comment]
Unfortunately we can't do anything about OEMs blocking certain APIs from us other than to let people know to double check app permissions and trying to talk to OEMs 🤷🏻

**Comment by burtek** [comment]
@alex-signal aware of that, but opened this as per [linked comment](https://github.com/signalapp/Signal-Android/issues/13325#issuecomment-2067022409)

> If any new issues with Bubbles arise, please file a new GitHub issue

**Comment by nicholas-signal** [comment]
I have had bubbles working successfully on my Samsung A51, A54, and S24 Ultra here in the USA. I have come to believe that the worldwide rollout is solely under Samsung's control.

**Comment by burtek** [comment]
Just for reference, May 2024 and June 2024 releases for A55 5G in Poland do not fix the issue. Samsung is middle-fingering us.

**Comment by burtek** [comment]
@alex-signal 

> Unfortunately we can't do anything about OEMs blocking certain APIs from us other than (...) trying to talk to OEMs

From https://github.com/signalapp/Signal-Android/issues/13325#issuecomment-2067022409 I understood it has already been discussed with Samsung and that they whitelisted Signal

> we have cooperated with Samsung to restore Bubble privileges to Signal

I'm just saying that seems to be a lie on their part as I don't believe they did that fully in all OS versions/for all devices... Is there any point of contact that we as users can try to talk to Samsung? Tried reporting this as issue from in-phone menu, but so far no luck (and no response)

**Comment by nicholas-signal** [comment]
Hello all,

I have received a request for logs from Samsung from one of the affected devices. Please follow the instructions Samsung has provided below so that the Samsung engineers can take a look:

> Please follow the given procedure to collect the logs:
> a) Dial *#9900#, Delete Dumstate/logcat 
> 
> b) Dial *[#9900](https://samsungdeveloper.zendesk.com/tickets/9900)#, Select Debug Level to MID. This restarts the device. 
> 
> c) Reproduce the issue (receive a message on the Signal app and check that bubbles are not working)
> 
> d) Again dial *[#9900](https://samsungdeveloper.zendesk.com/tickets/9900)# and press “Run Dumpstate/logcat”
> 
> e) When it is finished press “Copy to SD Card (include CP Ramdump) 
> 
> f) In the main (root) directory of the device file system, please look for the directory 'log'. That directory would contain the dupmstate log file generated, having the name format: dumpState_<DEVICE MODEL>_<TIMESTAMP>
> 
> A screen recording of the issue taken at the same time as recording the logs would be helpful as well.

Thank you for your patience!

**Comment by burtek** [comment]
Don't worry, I'm gonna share my logs tomorrow.
